definition-importer/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM openjdk:17-buster
 
 ENV FHIR_BASE=http://localhost:8080/fhir
+ENV FHIR_VERSION=dstu3
```

Comment:
`dstu3` should be `stu3`

Reply:
Unfortunately the hapi fhir server doesn't understand stu3, that's why I'm calling it dstu.

Reply:
OK. Can you add one comment in the Dockerfile to provide the list of available versions?

Reply:
Added it. Also in the readme